src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item Standard Report Type in GS for End User</shortDescription>
+        <value>Learn more about Standard Report Types in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypePriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Standard Report Type in GS for End User</shortDescription>
+        <value>Create Reports</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Standard Report Type in GS for End User</shortDescription>
+        <value>Create a Report from a Standard Report Type</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item View Dashboard in GS for End User</shortDescription>
+        <value>Dashboards pull data from one of more reports and display it side by side in the same window. NPSP includes three dashboards that you can use as is or customize.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item View Dashboard in GS for End User</shortDescription>
+        <value>Learn more about Dashboards in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item View Dashboard in GS for End User</shortDescription>
+        <value>View Dashboards</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item View Dashboard in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Title of View Dashboard checklist item in Get Started for End Users.